lib/ssh/src/ssh.hrl

```diff
@@ -514,7 +515,7 @@ protocol).
 Opaque types that define experimental options that are not to be used in
 products.
 """.
--doc(#{title => <<"Other data types">>}).
+-doc(#{title => <<"Other Data Types">>}).
```

Comment:
What is meant with opaque option? This does not make sense to me!

Reply:
you mean `opaque_` part in type names below?

Reply:
> And that is why we have some values that are only documented opaque as we want users to be able to match them for equality although we do think it is an API violation to pick them apart and do things with an individual part.

Testing (in-)equality of opaques _with the same name_ is okay in `master`, as it does not leak any information about the contents. You can start looking into making these types actually opaque now :-)